marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -296,6 +297,22 @@ private HubModuleManager getPropsMgr() {
         return entities;
     }
 
+    public HubEntity getEntity(String entityName) {
+        return getEntity(entityName, null);
+    }
+
+    public HubEntity getEntity(String entityName, String version) {
+        HubEntity entity = null;
+        for (HubEntity e: getEntities()) {
+            InfoType info = e.getInfo();
+            if (info.getTitle().equals(entityName) && (version == null || info.getVersion().equals(version))) {
```

Comment:
info.getTitle() can be null?

Reply:
Title is a required field for entities, so it should never be null.